docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/field-message/field-message.scss

```diff
@@ -0,0 +1,42 @@
+//
+// Field description component.
+//
+
+@use 'sass:color';
+
+@mixin _civictheme-background-color-opacity($opacity ,$color) {
+  background-color: rgba(color.red($color), color.green($color), color.blue($color), $opacity);
+}
+
+.civictheme-field-message {
+  $root: &;
+
+  display: block;
+  border-radius: $civictheme-border-radius;
+  padding: civictheme-space(0.5);
+
+  @include civictheme-typography('text-regular');
+
+  @each $type, $value in _civictheme-colors-all() {
+    &--#{$type} {
+      border: 1px solid civictheme-color($type);
```

Comment:
border and colour mapping must always be sitting separately - border with and style are applied outside of `.civictheme-theme-<theme>{}` rules

Reply:
Background color and border color are based on the type of message.  And opacity only if the theme is light.

Reply:
the opacity colour should be mapped through a variable